Mathlib/Data/Option/Defs.lean

```diff
@@ -57,3 +60,79 @@ def melim {α β : Type _} {m : Type _ → Type _} [Monad m] (x : m (Option α))
 /-- A monadic analogue of `Option.getD`. -/
 def mgetD {α : Type _} {m : Type _ → Type _} [Monad m] (x : m (Option α)) (y : m α) : m α :=
   melim x y pure
```

Comment:
And this one probably `getMD` (although I couldn't complain if it were `getDM`...?)